src/Symfony/Component/Lock/Store/DoctrineDbalStore.php

```diff
@@ -235,4 +243,21 @@ private function getCurrentTimestampStatement(): string
                 return (string) time();
         }
     }
+
+    /**
+     * Checks wether current driver supports table creating within transaction.
+     */
+    private function driverSupportsTableCreationInTransaction(): bool
+    {
+        $platform = $this->conn->getDatabasePlatform();
+
+        switch (true) {
+            case $platform instanceof \Doctrine\DBAL\Platforms\PostgreSQLPlatform:
```

Comment:
missing `PostgreSQL94Platform` have a look to `getCurrentTimestampStatement` for reference of existing platforms

Reply:
Added `PostgreSQL94Platform`.